src/solver/conv_winoRxS_f2x3.cpp

```diff
@@ -196,6 +196,11 @@ void PerformanceConfigConvBinWinogradRxSf2x3::EuristicInit(const ConvolutionCont
 {
     const auto n_inputs_per_group  = config.n_inputs / config.group_counts,
                n_outputs_per_group = config.n_outputs / config.group_counts;
+    if(config.group_counts == 1)
```

Comment:
Bugfix?

Reply:
performance fix 😅.
EuristicInit can't calculate fastest configuration in some cases. But this is fastest configuration  for all group_counts == 1.